static/app/components/events/autofix/autofixDrawer.spec.tsx

```diff
@@ -0,0 +1,99 @@
+import {AutofixDataFixture} from 'sentry-fixture/autofixData';
+import {AutofixStepFixture} from 'sentry-fixture/autofixStep';
+import {EventFixture} from 'sentry-fixture/event';
+import {GroupFixture} from 'sentry-fixture/group';
+import {ProjectFixture} from 'sentry-fixture/project';
+
+import {act, render, screen, userEvent, waitFor} from 'sentry-test/reactTestingLibrary';
+
+import {AutofixDrawer} from 'sentry/components/events/autofix/autofixDrawer';
+import {t} from 'sentry/locale';
+
+describe('AutofixDrawer', () => {
+  const mockEvent = EventFixture();
+  const mockGroup = GroupFixture();
+  const mockProject = ProjectFixture();
+
+  const mockAutofixData = AutofixDataFixture({steps: [AutofixStepFixture()]});
+
+  beforeEach(() => {
+    MockApiClient.clearMockResponses();
+  });
+
+  it('renders properly', () => {
+    MockApiClient.addMockResponse({
+      url: `/issues/${mockGroup.id}/autofix/`,
+      body: {autofix: mockAutofixData},
+    });
+
+    render(<AutofixDrawer event={mockEvent} group={mockGroup} project={mockProject} />);
+
+    expect(screen.getByText(mockGroup.shortId)).toBeInTheDocument();
+
+    expect(screen.getByText(mockEvent.id)).toBeInTheDocument();
+
+    expect(screen.getByRole('heading', {name: t('Autofix')})).toBeInTheDocument();
```

Comment:
don't translate in tests
```suggestion
    expect(screen.getByRole('heading', {name: 'Autofix'})).toBeInTheDocument();
```